.github/workflows/make_release.yml

```diff
@@ -89,7 +89,7 @@ jobs:
             artifact: Larwick_Overmap
             tileset_dir: gfx/Larwick_Overmap
             compose_args: --use-all
-          - name: MShockXotto+
+          - name: MshockXotto+
```

Comment:
I only realised after merging it but it's the `artifact` entry that needs changing here, see lines 133 - 134

Reply:
Sorry about that, turns out I am blind